projects/commudle-admin/src/app/feature-modules/community-control-panel/community-control-panel.module.ts

```diff
@@ -81,10 +80,5 @@ import { CommunityTeamComponent } from './components/community-team/community-te
     NbContextMenuModule,
     NbCheckboxModule,
   ],
-  entryComponents: [
-    CommunityFormsListStatsComponent,
-    CommunityFormsListActionsComponent,
-    // EmailerComponent
-  ],
```

Comment:
Why were these removed?

Reply:
- EntryComponents used to create NgFactories in order to render components were only necessary till version 8 of angular. - In the migration to 13, they were removed by default because the Ivy renderer makes the need to specify components in the entryComponents array unnecessary and stores all the information required to render a component.